packages/ketcher-react/src/script/ui/views/modal/components/Text/Text.module.less

```diff
@@ -0,0 +1,20 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+
+.text_area {
```

Comment:
please use camel case for naming

Reply:
I was basing my naming on PeriodTable.module.less, shall we refactor it as well?

Reply:
Refactoring of periodic table will be done later